data/json/construction.json

```diff
@@ -1019,6 +1019,21 @@
     "pre_special": "check_support",
     "post_terrain": "t_floor"
   },
+  {
+    "type": "construction",
+    "description": "Build Thatched Roof",
+    "category": "CONSTRUCT",
+    "required_skills": [ [ "fabrication", 3 ], [ "survival", 3 ] ],
```

Comment:
Where is the survivor learning thatching? It's a complex skill that requires understanding a lot about the performance of the reeds used. http://endeavourcentre.org/2017/04/thatch-roof-basics/ it takes years to learn to do thatch without it leaking and falling apart, it's not something you just figure out after a few days in the bush.

Reply:
What fabrication/survival levels would you suggest then? 7/7? (Note that it would be a lovely use for high level survival which is IIRC a little useless currently)